broadcaster/assets/js/app.js

```diff
@@ -1,6 +1,6 @@
 // If you want to use Phoenix channels, run `mix help phx.gen.channel`
 // to get started and then uncomment the line below.
-import "./user_socket.js"
+// import "./user_socket.js"
```

Comment:
Importing user_socket this way, we count our player panel into the viewers. It could also be tricky to make sure that  HTML chat elements are already loaded so I've renamed this file to `chat.js`, wrapped everything into a function and import this function in home.js.   

This way, chat.js is only loaded and used when needed.